CHANGELOG.md

```diff
@@ -1,3 +1,132 @@
+- OpenShift support is removed.  [#6539](https://github.com/kubermatic/kubermatic/issues/6539) 
+- Fix issue with gatekeeper not recognizing the AdmissionReview v1 version by changing the webhook to use v1beta1 [#6550](https://github.com/kubermatic/kubermatic/issues/6550) 
+- OpenStack: Add support for &#34;use-octavia&#34; setting in Cluster Openstack cloud specs. [#6565](https://github.com/kubermatic/kubermatic/issues/6565) 
+- Kubermatic Constraint Templates now support cluster filtering to allow users to manage the CT&#39;s and policies across different clusters more easily. The feature is available for Kubermatic EE.  [#6567](https://github.com/kubermatic/kubermatic/issues/6567) 
+- Fix KAS service port in Tunneling agent configuration. [#6569](https://github.com/kubermatic/kubermatic/issues/6569) 
+- Add support for Hetzner CCM [#6588](https://github.com/kubermatic/kubermatic/issues/6588) 
+- Increase default gatekeeper webhook timeout from 2 to 10 seconds, and added option in cluster settings to configure it. [#6590](https://github.com/kubermatic/kubermatic/issues/6590) 
+- Add support in Openstack datacenters to explicitly enable certain flavor types. [#6612](https://github.com/kubermatic/kubermatic/issues/6612) 
+- Extend SA controller for new SA type. [#6634](https://github.com/kubermatic/kubermatic/issues/6634) 
+- Fix the operator failing to reconcile the ValidatingWebhookConfiguration object for the cluster validation webhook [#6639](https://github.com/kubermatic/kubermatic/issues/6639) 
+- Provide the possibility of configuring leader election parameters for user cluster components. [#6641](https://github.com/kubermatic/kubermatic/issues/6641) 
+- Fix nodeport-proxy role used with LoadBalancer expose strategy. [#6646](https://github.com/kubermatic/kubermatic/issues/6646) 
+- Remove unused deprecated `certs` chart [#6656](https://github.com/kubermatic/kubermatic/issues/6656) 
+- Add `registry_mirrors` to Seed node settings [#6667](https://github.com/kubermatic/kubermatic/issues/6667) 
+- Make the `externalCloudProvider` feature gate impossible to disable once it has been enabled [#6668](https://github.com/kubermatic/kubermatic/issues/6668) 
+- Fix CE installer binary in EE downloads [#6673](https://github.com/kubermatic/kubermatic/issues/6673) 
+- Upgrad Gatekeeper from 3.1.0-beta-9 to 3.1.3. NOTICE: this change also moves the Gatekeeper deployment from the Seed to the User clusters. This means that the user clusters will need some additional resources to run the Gatekeeper Pods. Admins please refer to the upgrade guidelines in the documentation. [#6706](https://github.com/kubermatic/kubermatic/issues/6706) 
+- Change default gatekeeper webhook timeout to 3 sec [#6709](https://github.com/kubermatic/kubermatic/issues/6709) 
+- AWS Spot instances support in KKP [#6726](https://github.com/kubermatic/kubermatic/issues/6726) 
```

Comment:
Regarding `AWS Spot instances support in KKP` the release note hinted that the whole topic should have been done(including UI and the Spot monitoring controller), however, due to prio shifting, UI and the monitoring controller were not added. Thus I would suggest either remove this line, or(which I don't prefer) add this line instead:
```
Add spot instances as an option for the aws machines in the API 
```

Reply:
Removed